install/kubernetes/istio-one-namespace-auth.yaml

```diff
@@ -273,6 +273,7 @@ spec:
         args:
           - --configStoreURL=fs:///etc/opt/mixer/configroot
           - --configStore2URL=k8s://
+          - --traceOutput=http://zipkin.istio-system:9411/api/v1/spans
```

Comment:
For one namespace (shared cluster) setup, this needs to be customized.

Reply:
customized how?  Is zipkin itself going to live in a bunch of different namespaces ?

Reply:
@andraxylia or @ldemailly probably know how it gets customized. Yeah, zipkin gets deployed in application namespace in this case.

Reply:
it matches line 980 of the one-namespace file, currently.

Reply:
Longer term we'll move to use the same configmap as Envoy when determining what to do with traces.

Reply:
in shared env we do move all components into the 1 random namespace - we do that through some search/replace - I think @kyessenov's question is valid on whether that specific string will or will not get replaced too

Reply:
If we are deploying zipkin in `istio-system` namespace, then it is best to just use
`- --traceOutput=http://zipkin:9411/api/v1/spans`

Reply:
@ldemailly if you look at the PR, you will see that the tmpl file uses the same mechanism for replacement as all the rest of the things in istio-system.  This is no different than anything else.

@mandarjog why is that best?

Reply:
Location independent configuration is preferred if there is a choice. Leaving it out means that all these components are in the same namespace, and it does not matter which one it is.

Reply:
zipkin is deployed in istio-system, there is no need for special handling in this PR.